websockets/chat-broker/src/session.rs

```diff
@@ -137,13 +170,17 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for WsChatSession {
 
                         Some("/name") => {
                             if let Some(name) = command.next() {
-                                self.name = Some(name.to_owned());
+                                self.client_name = Some(name.to_owned());
                                 ctx.text(format!("name changed to: {}", name));
                             } else {
                                 ctx.text("!!! name is required");
                             }
                         }
 
+                        Some("/list-clients") => self.list_clients(ctx),
```

Comment:
I think new commands must be reflected in the `README.md`.

Also, it'd be nice to either update [original example](https://github.com/actix/examples/tree/master/websockets/chat) as well, or notice that this example is superior to the linked one somewhere near "This is a different implementation..." statement in `README.md`.

Reply:
ok, I did those things